README.md

```diff
@@ -56,15 +59,14 @@ If you've come here to help contribute - Thanks! Take a look at the [contributin
     - [.toBeBoolean()](#tobeboolean)
     - [.toBeTrue()](#tobetrue)
     - [.toBeFalse()](#tobefalse)
-  - [Date](#date)
-    - [.toBeDate()](#tobedate)
-    - [.toBeValidDate()](#tobevaliddate)
-    - [.toBeAfter(date)](#tobeafterdate)
-    - [.toBeBefore(date)](#tobebeforedate)
-    - Further proposals in [#117](https://github.com/jest-community/jest-extended/issues/117) PRs welcome
+  - [~~Date~~](#date)
```

Comment:
This formatting doesn't appear to be correct

<img width="1009" alt="Screen Shot 2021-06-28 at 11 08 42 AM" src="https://user-images.githubusercontent.com/2440089/123659979-3e7edb00-d801-11eb-9e89-d68c975bc2b9.png">

Reply:
This was auto generated by tooling, not by hand. I'm not sure what I should do to fix formatting.